website/docs/reference/node-selection/test-selection-examples.md

```diff
@@ -60,32 +62,38 @@ The "buildable" and "cautious" modes can be useful in environments when you're o
 
 </VersionBlock>
 
-<VersionBlock firstVersion="1.5" >
 
-These are the modes to configure the behavior when performing indirect selection (with `eager` as the default):
+<IndirSelect features={'/snippets/indirect-selection-definitions.md'}/>
 
-1. `eager` (default) - include ANY test that references the selected nodes
-1. `cautious` - restrict to tests that ONLY refer to selected nodes
-1. `buildable` -  restrict to tests that ONLY refer to selected nodes (or their ancestors)
-1. `empty` - restrict to tests that are only for the selected node and ignore all tests from the attached nodes 
 
-Note that test exclusion is always greedy: if ANY parent is explicitly excluded, the test will be excluded as well.
+<!--tabs for eager mode, cautious mode, empty, and buildable mode -->
+<!--Tabs for 1.5+ -->
 
-The "buildable", "cautious", and "empty" modes can be useful in environments when you're only building a subset of your DAG, and you want to avoid test failures in "eager" mode caused by unbuilt resources. (Another way to achieve this is with [deferral](/reference/node-selection/defer)).
+### Indirect selection examples
 
-</VersionBlock>
+<VersionBlock firstVersion="1.5">
 
-<!--tabs for eager mode, cautious mode, and buildable mode -->
+To visualize these methods, suppose you have `model_a`, `model_b`, and `model_c` and associated data tests. Here's what it will look like when you execute `dbt run` vs. `dbt build` with the various indirect selection modes:
 
-<VersionBlock lastVersion="1.3">
+<DocCarousel slidesPerView={1}>
 
-<Tabs queryString="indirect-selection-mode">
-<TabItem value="eager" label="Eager mode (default)">
+<Lightbox src src="/img/docs/reference/indirect-selection-dbt-run.png" width="85%" title="dbt run"/>
 
-By default, a test will run when ANY parent is selected; we call this "eager" indirect selection. In this example, that would include any test that references orders, even if it references other models as well.
+<Lightbox src src="/img/docs/reference/indirect-selection-eager.png" width="85%" title="Eager (default)"/>
 
-In this mode, any test that depends on unbuilt resources will raise an error.
+<Lightbox src src="/img/docs/reference/indirect-selection-cautious.png" width="85%" title="Cautious"/>
+
+<Lightbox src src="/img/docs/reference/indirect-selection-buildable.png" width="85%" title="Buildable"/>
+
+<Lightbox src src="/img/docs/reference/indirect-selection-empty.png" width="85%" title="Empty"/>
+
+</DocCarousel>
 
+<Tabs queryString="indirect-selection-mode">
+<TabItem value="eager" label="Eager mode (default)">
+
+For example, during the build process, any model that depends on the selected "orders" model or its dependent models will be built. Tests that refer to the selected "order" model or its dependent models will be executed during the testing process:
```

Comment:
This isn't accurate.

Here's a slight tweak that gets it back in line:
```suggestion
In this example, during the build process, any test that depends on the selected "orders" model or its dependent models will be executed, even if it depends other models as well.
```

Reply:
That's much clearer too! 🙇🏻‍♀️